sdk/identity/identity/samples/AzureIdentityExamples.md

```diff
@@ -255,6 +253,17 @@ function withDeviceCodeCredential() {
 }
 ```
 
+To authenticate a user through device code flow, use the following steps:
+
+1. Go to Azure Active Directory in Azure portal and find your app registration.
+2. Navigate to the **Authentication** section.
+3. Under **Suggested Redirected URIs**, check the URI that ends with `/common/oauth2/nativeclient`.
```

Comment:
These instructions seem to be out of date with the current portal experience. Now there seems to be a Platform configurations setting and you can add one or more configurations.

Reply:
What about these instructions:

> 1. Go to Azure Active Directory in Azure portal and find your app registration.
> 2. Navigate to the **Authentication** section.
> 3. Under **Advanced settings**, select `yes` on the option `Enable the following mobile and desktop flows`.